src/driver.ts

```diff
@@ -536,12 +536,14 @@ export abstract class CMakeDriver implements vscode.Disposable {
         if (gen.name.toLowerCase().startsWith('xcode') && platform === 'darwin') {
           return gen;
         }
-        vscode.window.showErrorMessage(`Unknown CMake generator "${gen.name}"`);
         continue;
       } else {
         return gen;
       }
     }
+    vscode.window.showErrorMessage(
+      `Not of the preferred cmake generator found.
+Please install or configure your preferred generator needs in the settings.json or cmake-kit.json.`);
```

Comment:
Try this:

`Unable to determine what CMake generator to use. Please install and configure a preferred generator, or update settings.json or your Kit configuration.`

---

I'd like to eventually have links in the error messages that jump to relevant user documentation.

Reply:
Hyperlinks do not work, but we could add an action.
![OpenDocumentation](https://user-images.githubusercontent.com/21360085/37736168-6f794eb6-2d50-11e8-8c55-42ced7e0a8a6.PNG)